# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoader.cpp
# Commit: 7d5683180ac3
# Full Hash: 7d5683180ac3f9e7cc96d68dde27b8b48642a7ac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r a48aff5f2555 -r 7d5683180ac3 dom/script/ModuleLoader.cpp
--- a/dom/script/ModuleLoader.cpp	Thu Feb 17 15:57:52 2022 +0000
+++ b/dom/script/ModuleLoader.cpp	Thu Feb 17 15:57:52 2022 +0000
@@ -5,12 +5,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "ScriptLoader.h"
-#include "ScriptLoadRequest.h"
 #include "ScriptTrace.h"
-#include "LoadedScript.h"
 #include "ModuleLoader.h"
-#include "ModuleLoaderBase.h"
-#include "ModuleLoadRequest.h"
 
 #include "jsapi.h"
 #include "js/ContextOptions.h"  // JS::ContextOptionsRef
@@ -20,6 +16,10 @@
 #include "js/PropertyAndElement.h"  // JS_DefineProperty
 #include "js/Realm.h"
 #include "js/SourceText.h"
+#include "js/loader/LoadedScript.h"
+#include "js/loader/ScriptLoadRequest.h"
+#include "js/loader/ModuleLoaderBase.h"
+#include "js/loader/ModuleLoadRequest.h"
 #include "xpcpublic.h"
 #include "GeckoProfiler.h"
 #include "nsIContent.h"
@@ -32,6 +32,7 @@
 #include "mozilla/LoadInfo.h"
 
 using JS::SourceText;
+using namespace JS::loader;
 
 namespace mozilla::dom {
 